libraries/featureflag/api/src/main/kotlin/io/element/android/libraries/featureflag/api/FeatureFlags.kt

```diff
@@ -16,21 +16,26 @@
 
 package io.element.android.libraries.featureflag.api
 
+/**
+ * To enable or disable a FeatureFlags, change the `defaultValue` value.
+ * It will impact all the build types.
+ */
 enum class FeatureFlags(
     override val key: String,
     override val title: String,
     override val description: String? = null,
-    override val defaultValue: Boolean = true
+    override val defaultValue: Boolean
```

Comment:
I'd keep the defaultValue to true, so it's enabled by default for debug

Reply:
I do not agree, for debug, you can enable it in the develop option.

Reply:
But I admit it can be an issue if the flags is impacting the FTUE flow...

Reply:
Yes of course, but defaultValue was more intended to be use as a fallback value. 
We should probably add more comment then, adding a reference to the StaticFeatureFlagProvider.

Reply:
Developers (or people enabling flags) do not have to know about `StaticFeatureFlagProvider` IMO.

Reply:
And IMO, it's the only file they have to know :D
We could add a ReleaseFeatureFlags file somewhere, and StaticFeatureFlagProvider would just read from it?

Reply:
Maybe it's a naming issue with `defaultValue`.
We can have a flag with `defaultValue = false`, and the feature enabled in `StaticFeatureFlagProvider`. To confusing IMO.